build-info-extractor-docker/src/main/java/org/jfrog/build/extractor/docker/DockerAgentUtils.java

```diff
@@ -0,0 +1,142 @@
+package org.jfrog.build.extractor.docker;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.BuildImageCmd;
+import com.github.dockerjava.api.model.AuthConfig;
+import com.github.dockerjava.core.DefaultDockerClientConfig;
+import com.github.dockerjava.core.DockerClientBuilder;
+import com.github.dockerjava.core.DockerClientConfig;
+import com.github.dockerjava.core.command.BuildImageResultCallback;
+import com.github.dockerjava.core.command.PullImageResultCallback;
+import com.github.dockerjava.core.command.PushImageResultCallback;
+import com.github.dockerjava.netty.NettyDockerCmdExecFactory;
+import org.apache.commons.lang.StringUtils;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Map;
+
+
+public class DockerAgentUtils {
```

Comment:
I think that class can be merged with DockerUtils.
It looks like there's no need to hold both classes.

Reply:
Renamed to `DockerJavaWrapper`